glslc/test/working_directory.py

```diff
@@ -207,11 +157,63 @@ class TestWorkDirCompileFileAbsolutePath(expect.ValidObjectFile):
     glslc_args = ['-c', '-working-directory=subdir', shader]
 
 
-@inside_glslc_testsuite('WorkDir')
-class TestWorkDirLinkFileAbsolutePath(expect.ValidNamedObjectFile):
-    """Tests -working-directory=<dir> when linking input file with absolute
-    path."""
+# The -working-directory flag should not affect the placement of the link file.
+# The following tests ensure that.
 
-    shader = FileShader('void main() {}', '.vert')
-    glslc_args = ['-working-directory=subdir', shader]
+class WorkDirDoesntAffectLinkedFile(expect.ValidNamedObjectFile):
+    """A base class for tests asserting that -workind-directory has no impact
```

Comment:
Typo:  "-working-directory".

More importantly, working-directory *does* affect where the output file goes.  We want to match clang behaviour.  (Try it.)
But this is a matter of incomplete feature, rather than this CL.